src/platform/nodejs/noopSender.ts

```diff
@@ -13,7 +12,7 @@ export class NoopSender implements Sender {
 
   send(payload: Envelope[], callback: SenderCallback): void {
```

Comment:
Why do you keep this sender?, is there any use for it?

Reply:
Deleting it